config.yaml

```
@@ -44,7 +46,11 @@ debug:
     archive_states: False
 
 env:
-  headless: True
+  headless: False
+  head_ids: #Which environment IDs to display in pyboy (env_id starts at 1 when multithreaded)
+    - 1
+    - 5
+  disable_ai_actions: False
```

```suggestion
  headless: True
```
We don't want these settings to be true during normal runs.